specification/hugr.md

```diff
@@ -1236,10 +1236,10 @@ The `Replace` method takes as input:
     $G$ and `SrcNode` in $\Gamma \setminus R$;
   - a list $\mu\_\textrm{out}$ of `NewEdgeSpec` which all have their `SrcNode`in
     $G$ and `TgtNode` in $\Gamma \setminus R$, where `TgtNode` and `TgtPos` describe
-    an existing incoming edge of that kind from a node in $R$.
+    an existing incoming edge of that kind from a node in $S\*$.
```

Comment:
These tweaks allow one to replace an edge into/out-of a part of the Hugr that you are keeping but moving to a new location (the old edge might be invalid if it were nonlocal, say)